dependencies/pom.xml

```diff
@@ -81,6 +81,12 @@
       <version>0.42</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.google.truth.extensions</groupId>
+      <artifactId>truth-java8-extension</artifactId>
```

Comment:
Extension for Java8's optional.
https://google.github.io/truth/extension

Reply:
This should probably become part of core truth in 1.0.

Reply:
I agree.